common/static/coffee/src/discussion/views/discussion_thread_edit_view.js

```diff
@@ -75,7 +81,9 @@
                         this.$('.edit-post-title').val('').attr('prev-text', '');
                         this.$('.edit-post-body textarea').val('').attr('prev-text', '');
                         this.$('.wmd-preview p').html('');
-                        postData.courseware_title = this.topicView.getFullTopicName();
+                        if (this.topicView) {
```

Comment:
Why are there 2 places where update postData gets set? Just trying to understand the existing code...